.github/file-filters.yml

```diff
@@ -16,17 +16,21 @@ yarn_lockfile: &yarn_lockfile
 eslint_config: &eslint_config
   - '.eslint*'
 
-frontend: &frontend
+frontend_src: &frontend_src
   - *yarn_lockfile
-  - *frontend_lintable
```

Comment:
This is my preferred approach, separate "lintable" from "src" and "all". Remove this filter and inline the pattern.

Rationale being that if we try and give lintable the same notion of "src" vs "all" we can end up with "lintable_src" and "lintable_test", which is a little fishy to me.

But happy to take